pkg/kp/rc.go

```diff
@@ -0,0 +1,14 @@
+package kp
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerStore interface {
+	Create(manifest pods.Manifest, nodeSelector, podSelector labels.Selector) (string, error)
```

Comment:
someone is going to mix up the order of the arguments `nodeSelector` and `podSelector` and it is going to suck. Any way to make better?